apps/roam/src/components/DiscourseContextOverlay.tsx

```diff
@@ -244,23 +420,27 @@ const DiscourseContextOverlay = ({
             {selectedPage && (
               <div className="mt-6">
                 <h3 className="mb-2 text-base font-semibold">
-                  Suggested Relationships
+                  Suggested Relationships (Ranked by HyDE)
```

Comment:
This will likely not mean anything to majority of end users. (and could just confuse)